python/lsst/afw/image/apCorrMap/apCorrMap.cc

```diff
@@ -41,36 +42,43 @@ namespace {
 
 using PyApCorrMap = py::class_<ApCorrMap, std::shared_ptr<ApCorrMap>, typehandling::Storable>;
 
-PYBIND11_MODULE(apCorrMap, mod) {
-    py::module::import("lsst.afw.table.io");
-    py::module::import("lsst.afw.typehandling");
+void wrapApCorrMap(lsst::utils::python::WrapperCollection &wrappers) {
+     wrappers.wrapType(PyApCorrMap(wrappers.module, "ApCorrMap"), [](auto &mod, auto &cls) {
 
-    /* Declare CRTP base class. */
-    PyApCorrMap cls(mod, "ApCorrMap");
+         cls.def(py::init<>());
+
+         table::io::python::addPersistableMethods<ApCorrMap>(cls);
 
-    /* Constructors */
-    cls.def(py::init<>());
+         /* Operators */
+         cls.def("__imul__", &ApCorrMap::operator*=);
+         cls.def("__itruediv__", &ApCorrMap::operator/=);
 
-    table::io::python::addPersistableMethods<ApCorrMap>(cls);
+         /* Members */
+         cls.def("get", &ApCorrMap::get);
+         cls.def("set", &ApCorrMap::set);
+         cls.def("items", [](ApCorrMap const &self) { return py::make_iterator(self.begin(), self.end()); },
+                 py::keep_alive<0, 1>());
+         // values, keys, and __iter__ defined in apCorrMap.py
 
-    /* Operators */
-    cls.def("__imul__", &ApCorrMap::operator*=);
-    cls.def("__itruediv__", &ApCorrMap::operator/=);
+         cls.def("__len__", &ApCorrMap::size);
+         cls.def("__getitem__", &ApCorrMap::operator[]);
+         cls.def("__setitem__", &ApCorrMap::set);
+         cls.def("__contains__",
+                 [](ApCorrMap const &self, std::string name) { return static_cast<bool>(self.get(name)); });
 
-    /* Members */
-    cls.def("get", &ApCorrMap::get);
-    cls.def("set", &ApCorrMap::set);
-    cls.def("items", [](ApCorrMap const& self) { return py::make_iterator(self.begin(), self.end()); },
-            py::keep_alive<0, 1>());
-    // values, keys, and __iter__ defined in apCorrMap.py
+     });
+}
+
+PYBIND11_MODULE(apCorrMap, mod) {
+    lsst::utils::python::WrapperCollection wrappers(mod, "lsst.afw.image.apCorrMap");
+    wrappers.addInheritanceDependency("lsst.afw.table.io");
```

Comment:
I think this can be a signature dependency.

Reply:
Fixing in image ticket.